.github/workflows/ci.yaml

```diff
@@ -60,28 +60,44 @@ jobs:
         pip install -r tests/requirements-test.txt
         python -m pytest -v -p no:warnings
         deactivate
+  deployment-tests:
+    runs-on: self-hosted
+    strategy:
+      matrix:
+        profile: ["local-code", "latest-release"]
+      fail-fast: true
+    steps:
+    - uses: actions/checkout@v2
     - name: Setup Cluster
       run: |
         kind delete cluster 2>/dev/null
         kind create cluster
         kubectl get nodes
     - name: Deploy From Source
+      if: ${{ matrix.profile != 'latest-release' }}
```

Comment:
`matrix.profile != 'latest-release'`  ->>  `matrix.profile == 'local-code'`

Reply:
I was trying to leave it open to add more skaffold profiles for testing that would all use this step (maybe for deploying and testing the monolith?)